internal/auth/repository/eventsourcing/eventstore/auth_request.go

```diff
@@ -1615,8 +1615,6 @@ func userSessionByIDs(ctx context.Context, provider userSessionViewProvider, eve
 			if userAgentID != agentID {
 				continue
 			}
-		case user_repo.UserRemovedType:
```

Comment:
I think if we remove the case here we should handle the event in the `session.AppendEvent` and set the session to inactive. Or do't we need to make sure a session of the previous user is still usable?

Reply:
that's exactly what i did

Reply:
oh wow collapsed the last file too early 🤣